lib/libutils/ext/ftrace/ftrace.c

```diff
@@ -10,17 +10,31 @@
  * profiled too.
  */
 
-#include <arm_user_sysreg.h>
```

Comment:
You are doing more than needed in this patch. I'd rather have it only move the code, any further change to support kernel mode in addition to user space should come in a different commit.

Reply:
Okay.